paddle/phi/api/yaml/op_compat.yaml

```
@@ -1254,6 +1254,15 @@
       data_type : float
       support_tensor : true
 
+- op : fused_adam
```

```suggestion
- op : fused_adam_(fused_adam)
```
这样就会把旧IR的fused_adam算子映射到PIR的fused_adam_